compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/canonicalize_doubly_strided_op.mlir

```diff
@@ -0,0 +1,250 @@
+// RUN: iree-opt --pass-pipeline="builtin.module(func.func(iree-amdaie-canonicalize-doubly-strided-op,canonicalize))" %s | FileCheck %s
+
+// CHECK-LABEL: func.func @circular_dma_cpy_nd_source_target
```

Comment:
I dont see in this test what canonicalization happened as the checks are same as the input IR?

Reply:
Yeah, I am just verifying here that source/target is still correct after the transformation.

Reply:
It's good to add a comment saying the outputs are expected to be the same.

Reply:
Sure, will add it

Reply:
Addded the comment